apps/emqx/test/emqx_channel_hookpoints_SUITE.erl

```diff
@@ -1,8 +1,8 @@
 %%--------------------------------------------------------------------
-%% Copyright (c) 2018-2025 EMQ Technologies Co., Ltd. All Rights Reserved.
+%% Copyright (c) 2025 EMQ Technologies Co., Ltd. All Rights Reserved.
```

Comment:
Why change copyright years?

Reply:
The file appeared in 2023, so 2018 was a typo. Changed to 2023-2025